package com.app.network.helper;

import java.lang.System;

@kotlin.Metadata(mv = {1, 8, 0}, k = 1, d1 = {"\u0000\u0018\n\u0002\u0018\u0002\n\u0002\u0010\u0000\n\u0002\b\u0002\n\u0002\u0010\u000e\n\u0000\n\u0002\u0010\u0003\n\u0000\b\u00c7\u0002\u0018\u00002\u00020\u0001B\u0007\b\u0002\u00a2\u0006\u0002\u0010\u0002J\u000e\u0010\u0003\u001a\u00020\u00042\u0006\u0010\u0005\u001a\u00020\u0006\u00a8\u0006\u0007"}, d2 = {"Lcom/app/network/helper/Error;", "", "()V", "handleException", "", "throwable", "", "data_debug"})
public final class Error {
    @org.jetbrains.annotations.NotNull
    public static final com.app.network.helper.Error INSTANCE = null;
    
    private Error() {
        super();
    }
    
    @org.jetbrains.annotations.NotNull
    public final java.lang.String handleException(@org.jetbrains.annotations.NotNull
    java.lang.Throwable throwable) {
        return null;
    }
}